Author: Andrew Lee <andrew@linux.org.tw> <andrew@linux.org.tw>
Description: We don't need to run make in debian/ and mvsc/ subdirectory. 
No description.
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = m4 src stardict-plugins tests pixmaps po help data doc debian mvsc
+SUBDIRS = m4 src stardict-plugins tests pixmaps po help data doc
 ACLOCAL_AMFLAGS = -I m4
 
 EXTRA_DIST = \
--- a/configure.in
+++ b/configure.in
@@ -442,8 +442,6 @@
 stardict-withoutgnome.spec
 data/Makefile
 doc/Makefile
-debian/Makefile
-mvsc/Makefile
 po/Makefile.in
 m4/Makefile
 src/Makefile
--- a/autogen.sh
+++ b/autogen.sh
@@ -47,4 +47,3 @@
 autoconf || exit;
 echo "Running automake ...."
 automake || exit;
-./configure --prefix=/usr --sysconfdir=/etc --mandir=/usr/share/man --disable-deprecations $@
